metadata-ingestion/src/datahub/configuration/source_common.py

```diff
@@ -54,6 +54,13 @@ class DatasetSourceConfigMixin(PlatformInstanceConfigMixin, EnvConfigMixin):
     """
 
 
+class LowerCaseDatasetUrnConfigMixin(ConfigModel):
+    convert_urns_to_lowercase: bool = Field(
+        default=False,
+        description="Whether to convert dataset urns to lowercase.",
```

Comment:
Question: This property isn't specific to just dataset urns, is it?

Reply:
currently, it is only lowercase dataset urns

Reply:
Is that a design decision? What is the reasoning behind it?